camel/toolkits/github_toolkit.py

```diff
@@ -71,6 +72,63 @@ def summary(self) -> str:
         )
 
 
+@dataclass
+class GithubPullRequestDiff:
+    r"""Represents a single diff of a pull request on Github."""
+
+    def __init__(self, filename: str, patch: str) -> None:
+        r"""Initialize a GithubPullRequestDiff object.
+
+        Args:
+            filename (str): The name of the file that was changed.
+            patch (str): The diff patch for the file.
+        """
+        self.filename = filename
+        self.patch = patch
+
+    def summary(self) -> str:
```

Comment:
Maybe you can override `__str__(self):` and you can just use str(GithubPullRequestDiff) to get the summary